common-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/common/client/model/LoginModel.java

```diff
@@ -364,11 +364,11 @@ public boolean userHasPermissionToGetRmThreadDump() {
         return sessionPermissions.get(PERMISSION_RM_GET_RM_THREAD_DUMP);
     }
 
-    public boolean userHasPermissionToGetInfrasToPoliciesMapping() {
+    public boolean userDoesNotHavePermissionToGetInfrasToPoliciesMapping() {
```

Comment:
why the name of the function changed to the opposite sense without changing the implementation ?

Reply:
I will change the implementation